wpiutil/src/main/java/edu/wpi/first/wpiutil/RuntimeDetector.java

```diff
@@ -128,28 +128,12 @@ public static boolean isRaspbian() {
     }
   }
 
-  /** check if os is bionic aarch64.
+  /** check if architecture is aarch64.
    *
-   * @return if os is bionic aarch64
+   * @return if architecture is aarch64
    */
-  public static boolean isAarch64Bionic() {
```

Comment:
We should deprecate and delegate 🙂 

Cost is low

Reply:
All these functions will actually be going away soon anyway.